src/main/java/org/pesmypetcare/webservice/service/FreqWashServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package org.pesmypetcare.webservice.service;
+
+
+import org.pesmypetcare.webservice.dao.FreqWashDao;
+import org.pesmypetcare.webservice.entity.FreqWashEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class FreqWashServiceImpl implements FreqWashService {
+    @Autowired
+    private FreqWashDao freqWashDao;
+
+
+    @Override
+    public void createFreqWash(String owner, String petName, String date, FreqWashEntity freqWashEntity) {
+        freqWashDao.createFreqWash(owner, petName, date, freqWashEntity);
+    }
+
+    @Override
+    public void deleteAllFreqWashs(String owner, String petName) throws DatabaseAccessException {
+        freqWashDao.deleteAllFreqWashs(owner, petName);
+    }
+
+    @Override
+    public void deleteFreqWashByDate(String owner, String petName, String petDate) throws DatabaseAccessException {
+        freqWashDao.deleteFreqWashByDate(owner, petName, petDate);
+    }
+
+    @Override
+    public FreqWashEntity getFreqWashByDate(String owner, String petName, String petDate) throws DatabaseAccessException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 122).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5400494)